scramjet/streams.py

```diff
@@ -309,7 +308,7 @@ async def consume():
                 partial = chunks[-1]
 
             log(new_stream, f'leftover: {tr(partial)}')
-            if partial:
+            if partial: # pragma: no cover
```

Comment:
Could you add some comment in the code why it was ignored? :thinking:

Reply:
I will. Somehow pytest can't resolve it properly and claims that we are not able to go to lines after, cause partial is always True. Which is of course wrong behaviour from pytest. I don't have any idea how to resolve it better, so I've added #pragma  comment.